### DISCUSSION TITLE
[question] Possible to override background color of active taskbar tile?

### DISCUSSION THREAD

**Discussion by earshinov** [valinet/ExplorerPatcher#3261] [General]
Hey guys! Sorry, I am sure that this question must have already come up, but couldn't find it.

![image](https://github.com/valinet/ExplorerPatcher/assets/52021/21219c5c-8a7a-4a4d-aaa2-97ecee9db88f)

I am tired of straining my eyes to locate the currently active window on the taskbar 😩 Is there any way I could change either the active tile background or the background of taskbar in isolation from the active tile? If Explorer Patcher can't help me with that, any pointers are welcome.

I am also not happy with the tiny blue line underlying windows that are currently open (not minimized), but in comparison with the above this issue pales into insignificance.

**Comment by pyrates999** [comment]
Look at the accessibility settings that windows 11 provides: https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696

**Reply by earshinov** [reply]
Thanks! But a contrast theme is too big of a change for me.

![image](https://github.com/valinet/ExplorerPatcher/assets/52021/70dcae51-aae7-4730-bddc-1caf85f09426)

I am fairly happy with how Windows looks (given that I have an old-style taskbar thanks to ExplorerPatcher), the active taskbar tile being pretty much the only issue.

**Reply by pyrates999** [reply]
Try this: https://support.microsoft.com/en-us/windows/change-colors-in-windows-d26ef4d6-819a-581c-1581-493cfcc005fe

**Reply by earshinov** [reply]
@pyrates999 , thanks! I see that with transparency enabled the situation is better. I might stick to it unless there is a better option.

**Comment by Iowerth** [comment]
Same situation. Seeing I'm not the one with this problem. Wish to change background color of active tile and line color underneath it.

**Comment by pyrates999** [comment]
here's where you can change the accent colour:

![image](https://github.com/valinet/ExplorerPatcher/assets/106135298/c3bf0e8a-5c6f-4bc0-905f-88abc62c0965)

**Reply by Iowerth** [reply]
Thanks but it is not what I looking for.
